c_pal_ll/linux/devdoc/sync_linux_requirements.md

```diff
@@ -30,6 +33,24 @@ MOCKABLE_FUNCTION(, WAIT_ON_ADDRESS_RESULT, wait_on_address, volatile_atomic int
 
 **SRS_SYNC_LINUX_43_004: [** Otherwise, `wait_on_address` shall return `WAIT_ON_ADDRESS_ERROR`.**]**
 
+## wait_on_address_64
+
+```c
+MOCKABLE_FUNCTION(, WAIT_ON_ADDRESS_RESULT, wait_on_address_64, volatile_atomic int64_t*, address, int64_t, compare_value, uint32_t, timeout_ms)
+```
+
+**SRS_SYNC_LINUX_05_001: [** `wait_on_address_64` shall initialize a `timespec` struct with `.tv_nsec` equal to `timeout_ms* 10^6`. **]**
+
+**SRS_SYNC_LINUX_05_002: [** `wait_on_address_64` shall call `syscall` to wait on value at `address` to change than the one provided in `compare_value`. **]**
+
+**SRS_SYNC_LINUX_05_003: [** `wait_on_address_64` shall return true if the value at `address` changes than value at `compare_value`. **]**
```

Comment:
>at `address` changes than value at `compare_value [](http://example.com/codeflow?start=81&length=49)

Nit: awkward wording. Consider changing to "at `address` changes to a value different from `compare_value`"

Reply:
Fixed